examples/input/mouse_grab.rs

```diff
@@ -0,0 +1,26 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_system(grab_mouse)
+        .run();
+}
+
+// This system grab the mouse when you press the left mouse button and release
```

Comment:
```suggestion
// This system grabs the mouse when you the left mouse button is pressed and releases
```

Reply:
```suggestion
// This system grabs the mouse when the left mouse button is pressed
// and releases it when the escape key is pressed
```

IMO splitting this across two lines to match the sentence structure is clearer.